packages/block-editor/src/components/block-canvas/style.scss

```diff
@@ -2,12 +2,11 @@ iframe[name="editor-canvas"] {
 	width: 100%;
 	height: 100%;
 	display: block;
-}
-
-iframe[name="editor-canvas"]:not(.has-editor-padding) {
 	background-color: $white;
 }
 
 iframe[name="editor-canvas"].has-editor-padding {
-	padding: $grid-unit-60 $grid-unit-60 0;
+	border-width: $grid-unit-60 $grid-unit-60 0;
+	border-style: solid;
+	border-color: $gray-900; // Matches .edit-post-visual-editor and .edit-site-visual-editor backgrounds.
```

Comment:
Just a tiny note that @noisysocks has a PR that's looking to change the background color over in #58528. To avoid any clashes, whichever PR lands first will likely want to rebase and update to match the other one for consistency 😄

Reply:
Thanks for the heads up @andrewserong 

Give that PR touches both post and site editors, perhaps this bug might be better addressed in that PR? Or at least after it's merged 😄

Reply:
Nah keep em separate, it's fine.